src/services/users.js

```diff
@@ -35,9 +37,24 @@ module.exports = (env, jwt, database, sendEmail, mediaClient, validations) => {
     return jwt.sign({ code: random() }, { expiresIn: '24h' })
   }
 
+  let whiteListedDomains
+  try {
+    whiteListedDomains = env('WHITELISTED_DOMAINS').split(',').map(str => str.toLowerCase().trim())
+  } catch (e) {
+    console.log(`WHITELISTED_DOMAINS not set as comma delimited string of email addresses properly, error: ${e.stack}`)
+  }
+
+  const isDomainAuthorized = domain => !whiteListedDomains.length || whiteListedDomains.includes(domain)
```

Comment:
Please add a comment explaining this line -- at first blush, I thought "shouldn't that be `whiteListedDomains.length &&`, so what it's accomplishing is not as self-evident as I'd like.